Makefile

```diff
@@ -1,7 +1,9 @@
 all: install
 install: clean
+	#on python 2.6, pre-install last supported version of numpy
+	python --version 2>&1 >/dev/null | grep '2.6' && pip install 'numpy==1.10.0' || true
 	python setup.py sdist
-	pip install dist/*.tar.gz --upgrade
+	pip install dist/*.tar.gz --upgrade --upgrade-strategy=only-if-needed
```

Comment:
Yeah, my preference is to run against latest versions, so that we can immediately detect problems (like what happened now with the numpy / 2.6 problem)